src/shared/components/mutationMapper/MutationMapperStore.ts

```diff
@@ -62,6 +62,8 @@ import {
 
 export interface IMutationMapperStoreConfig {
     filterMutationsBySelectedTranscript?: boolean;
+    // TODO make ptmSource part of AppConfig? (IMutationMapperConfig)
+    ptmSource?: string;
```

Comment:
I'm thinking since we are supporting two sources, should we add an info icon or indicator to show which source of data you are seeing?

Reply:
Good point. I guess we can also update the existing PTM info tooltip to indicate the PTM data source.